src/components/screens/monitor/delegates/forgingDetails.test.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import delegates from '../../../../../test/constants/delegates';
+import ForgingDetails from './forgingDetails';
+
+describe('Forging Details', () => {
+  let wrapper;
+  let props = {
+    t: key => key,
+    sortDirection: 'asc',
+    networkStatus: {
+      data: {
+        supply: 9000000000000000000,
+      },
+    },
+    delegates: {
+      isLoading: true,
+      data: delegates.map((delegate, i) => ({
+        ...delegate,
+        forgingTime: i,
```

Comment:
💡 To make it more realistic You could use:
```suggestion
        forgingTime: moment().add(i * 10, 'seconds'),
```
https://momentjs.com/docs/#/manipulating/add/
 and mayby also shuffle the whole array with `.sort(() => Math.random() - 0.5)`
https://javascript.info/task/shuffle

Reply:
Thank you, Vit. Much better